packages/charts/backend/sirius-components-collaborative-charts/src/main/java/org/eclipse/sirius/components/collaborative/charts/HierarchyEventProcessorFactory.java

```diff
@@ -46,24 +46,25 @@ public HierarchyEventProcessorFactory(IRepresentationSearchService representatio
     }
 
     @Override
-    public boolean canHandle(IRepresentationConfiguration configuration) {
-        return configuration instanceof HierarchyConfiguration;
+    public boolean canHandle(String representationId) {
+        return this.representationSearchService.existByIdAndKind(representationId, IRepresentation.KIND_PREFIX + "?type=ForceDirectedTree") ||
+                this.representationSearchService.existByIdAndKind(representationId, IRepresentation.KIND_PREFIX + "?type=TreeMap") ||
+        this.representationSearchService.existByIdAndKind(representationId, IRepresentation.KIND_PREFIX + "?type=ZoomableCirclePacking");
```

Comment:
The type is `siriusComponents://representation?type=Hierarchy`, this is why your test is failing

Reply:
Well, it's one of the reason

Reply:
`org.eclipse.sirius.web.services.hierarchy.HierarchyDescriptionProvider#KIND` is not right

Reply:
Yep that's it, by changing this invalid test value to something like `ForceDirectedTree` the test is now working. Can you update it to let me perform a final review and merge the PR? Thanks :)

Reply:
Thanks, it was indeed an issue with `org.eclipse.sirius.web.services.hierarchy.HierarchyDescriptionProvider#KIND` I did not revert back the file when I tried at first to change the KIND for `Hierarchy `

Reply:
I'll see if I can prevent the triple requests to the database